.github/workflows/primer_comment.yaml

```diff
@@ -103,8 +103,9 @@ jobs:
           . venv/bin/activate
           python tests/primer/__main__.py compare \
           --commit=${{ github.event.workflow_run.head_sha }} \
-          --base-file=output_${{ steps.python.outputs.python-version }}_main.txt \
-          --new-file=output_${{ steps.python.outputs.python-version }}_pr.txt
+          --base-file=output_${{ steps.python.outputs.python-version }}_main_BATCHIDX.txt \
```

Comment:
Have you tested that the downloading of artifacts and unzipping works? As in, do the different batch jobs get zipped into one single file?

Reply:
The 4 files get [zipped up](https://github.com/jacobtylerwalls/pylint/actions/runs/5003019925) into one .zip, and when you download it you can examine the four files.